src/rmkit/input/events.cpy

```diff
@@ -133,6 +151,30 @@ namespace input:
               self.lifted = true
 
           break
+        case ABS_MT_WIDTH_MAJOR:
+          slots[slot].width_major = data.value
+          break
+        case ABS_MT_WIDTH_MINOR:
+          slots[slot].width_minor = data.value
+          break
+        case ABS_MT_TOUCH_MAJOR:
+          slots[slot].size_major = data.value
+          break
```

Comment:
from the docs I read, it's legal to omit MINOR if it's the same as MAJOR. I'm not sure if rm does that, but we might be able to copy this value to size_minor if it's set to -1 or something